bitnami/tomcat/templates/statefulset.yaml

```diff
@@ -27,20 +27,23 @@ spec:
       annotations: {{- include "common.tplvalues.render" (dict "value" .Values.podAnnotations "context" $) | nindent 8 }}
       {{- end }}
     spec: {{- include "tomcat.pod" . | nindent 6 }}
-  {{- if and .Values.persistence.enabled }}
   volumeClaimTemplates:
   - metadata:
       name: data
     spec:
-      accessModes: {{ .Values.persistence.accessModes }}
-      storageClassName: {{ .Values.persistence.storageClassName }}
+      accessModes:
+      {{- range .Values.persistence.accessModes }}
+      - {{ . | quote }}
+      {{- end }}
       resources:
         requests:
-          storage: {{ .Values.persistence.size }}
+          storage: {{ .Values.persistence.size | quote }}
+      {{- include "common.storage.class" (dict "persistence" .Values.persistence "global" .Values.global) | nindent 6 }}
+      {{- with .Values.persistence.selectorLabels }}
```

Comment:
I would like to further understand this change. Why is the `include` not enough?

Reply:
@marcosbc I fixed the typo in README